src/locales/fr/menu.ts

```diff
@@ -44,10 +44,114 @@ export const menu: SimpleTranslationEntries = {
   "score": "Score",
   "wave": "Wave",
   "loading": "Chargement…",
-  "loadingAsset": "Loading asset: {{assetName}}",
+  "loadingAsset": "Chargement des ressources: {{assetName}}",
```

Comment:
```suggestion
  "loadingAsset": "Chargement de la ressource: {{assetName}}",
```
Loading is carried out one resource at a time, in singular